source/dxgi/dxgi.cpp

```diff
@@ -126,17 +237,23 @@ static void init_reshade_runtime_d3d(T *&swapchain, unsigned int direct3d_versio
 	}
 	else if (direct3d_version == 12)
 	{
-		if (com_ptr<IDXGISwapChain3> swapchain3; SUCCEEDED(swapchain->QueryInterface(&swapchain3)))
+		if (com_ptr<IDXGISwapChain3> swapchain4; SUCCEEDED(swapchain->QueryInterface(&swapchain4)))
```

Comment:
There was no change here, so the variable name change is probably unintentional.

Reply:
Yes, i missed it, thks. Will fix it as soon as i can.